app/templates/users/show/index.hbs

```diff
@@ -1,10 +1,19 @@
 <div class="content">
   <div class="row row-chart">
-    <BarChart @data={{model.doiCount}} @label="DOI" @model={{model}} @link="users.show.dois" @summarize={{true}} />
-    <DonutChart @data={{model.resourceTypeCount}} @label="DOI" @model={{model}} @link="users.show.dois" />
+    {{#if (can "update user")}}
```

Comment:
Show the same chart (by publication year) to all users.

Reply:
already changed